screen-contributors/src/main/java/com/ivy/contributors/ContributorsDataSource.kt

```diff
@@ -35,4 +46,16 @@ class ContributorsDataSource @Inject constructor(
             null
         }
     }
+
+    suspend fun fetchIvyWalletRepositoryInfo(): IvyWalletRepositoryInfo? {
```

Comment:
Rename to `fetchRepositoryInfo` after you've done the above file rename, the word "ivyWallet" will already be in the class name